Ryoung27/redditClone
# added likes endpoints

steps to test
pull down code

git fetch --all
git checkout ryAddLikeEndpoints
make migrations
python manage.py makemigrations
python manage.py migrate
start server with python manage.py runserver
Check browsable API and make sure a likes was created.
